Makefile

```diff
@@ -0,0 +1,19 @@
+install_packages:
+	poetry install
+
+sort_imports:
+	isort .
+
+format:
+	black .
+
+lint:
+	flake8 .
+
+test:
+	pytest tests/
+
+precommit:
+	pre-commit install && pre-commit
```

Comment:
This does not work. Since the package is installed with Poetry, it has to run through Poetry:

```bash
poetry run pre-commit install && poetry run pre-commit run --all-files
```

Reply:
done